Podfile

```diff
@@ -38,11 +38,14 @@ target 'WooCommerce' do
   pod 'Gridicons', '~> 1.0'
 
   # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
-  pod 'WordPressAuthenticator', '~> 1.34.0-beta'
+  pod 'WordPressAuthenticator', '~> 1.34.0'
   # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :commit => ''
   # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => ''
   # pod 'WordPressAuthenticator', :path => '../WordPressAuthenticator-iOS'
 
+  # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
+  pod 'WordPressKit', '~> 4.25.0'
```

Comment:
Added this explicitly to ensure that the client app picks up the non-beta version of WordPressKit. Otherwise `WordPressKit` is only pulled as a transitive dependency from `WordPressAuthenticator`, which declares that dependency in its `WordPressAuthenticator.podspec` loosely, allowing beta versions, while in the client apps we want to ensure that we use a stable one.

This is the same trick that we've been using for a while in WordPress' `Podfile` – explicit declaration of `WordPressKit` to ensure non-beta dependency.

It never seem to have been necessary in Woo before because we were lucky enough to always do the `pod install` during release cute… right after a stable version of our dependencies (incl WordPressKit) were pushed to trunk and to CDN, but before any new beta was pushed. But this time around, and because the CDN took >1h30 to resync with master, `WordPressKit 4.26.0-beta.1` was already available on CDN by the time `WordPressAuthenticator 1.34.0` was finally synced in the CDN, i.e by the time our `pod install --repo-update` finally found it, which means that this time around, it picked the next beta of WPKit that was already available instead of the latest stable version pushed only minutes before it.